lib/loga/sidekiq/job_logger.rb

```diff
@@ -0,0 +1,54 @@
+module Loga
+  module Sidekiq
+    class JobLogger
+      include Loga::Utilities
+
+      EVENT_TYPE = 'sidekiq'.freeze
+
+      attr_reader :started_at, :data
+
+      def initialize
+        @started_at = Time.now
+        @data = {}
+      end
+
+      def call(item, _queue)
+        yield
+        data['status'] = 'success'
+      rescue Exception # rubocop:disable Lint/RescueException
+        data['status'] = 'failure'
+        raise
+      ensure
+        assign_data(item)
+        send_message
+      end
+
+      private
+
+      def assign_data(item)
+        data['created_at']  = item['created_at']
+        data['enqueued_at'] = item['enqueued_at']
+        data['jid']         = item['jid']
+        data['queue']       = item['queue']
+        data['retry']       = item['retry']
+        data['params']      = item['args']
+        data['class']       = item['class']
+        data['duration']    = duration_in_ms(started_at)
+      end
+
+      def send_message
+        event = Event.new(data: data, message: data['status'], type: EVENT_TYPE)
```

Comment:
The message key here is meant to summarise what happened. I think the following would read nicely
`<class> with id <jid> executed in <duration>ms`

I don't think we need to include the status because the log level will provide that information.